Issue #24 adding instructions for how to add a jar file (#44)
comp129/comp55: docs/labs/Supplemental/Installing_Jars_Into_Eclipse.md

# How to Add the ACM.jar to Your Build Path

If you wish to use the ACM library, it is not as simple as doing a `#include "acm.jar"`.
In order to use the library, you have to manually add it.

## Instructions

> Step 1: Import the `acm.jar` into your program (the **IntroACMLab.zip**)\
> Step 2: Right click your project\
> Step 3: Click the Properties tab towards the buttom\
> Step 4: Click on `Java Build Path`\
> Step 5: Click the `Add JARS` button\
> Step 6: Locate the JAR file and double click it\
> Step 7: Click `Apply and Close` to enable the just added JAR file\
> Step 8: Try to run your code

Reviewer: ```suggestion
1. Import the *acm.jar* into your program (the **IntroACMLab.zip**)
2. Right click your project
3. Click the Properties tab towards the button
4. Click on ***Java Build Path***
5. Click the ***Add JARs...*** button
6. Locate the JAR file and double-click it
7. Click ***Apply and Close*** to enable the just added JAR file
8. Try to run your code

```

![20230309_003217](https://user-images.githubusercontent.com/72991689/223967170-8119d305-4c50-4b69-856d-7e1578db9335.gif)
Note: In the gif, there is an error in the console in Eclipse. This is due to how
I tried to run the program without the JAR file in my build path.

> After enabling the JAR, I am able to run my program without any errors

Reviewer: ```suggestion
After enabling the JAR, I can run the program without any errors
```

> After enabling the JAR, I am able to run my program without error

Reviewer: ```suggestion
After enabling the JAR, I am able to run my program without any errors

```
